tests/test_playbooks/content_view_filter.yml

```diff
@@ -6,36 +6,69 @@
   vars_files:
     - vars/server.yml
   tasks:
-    - include_tasks: tasks/organization.yml
+    - name: "Create Test Organization"
+      ansible.builtin.include_tasks: tasks/organization.yml
       vars:
         organization_state: present
-    - include_tasks: tasks/product.yml
+
+    - name: "Create Test Product"
+      ansible.builtin.include_tasks: tasks/product.yml
       vars:
         product_state: present
-    - include_tasks: tasks/repository.yml
+
+    - name: "Create Test Repository"
+      ansible.builtin.include_tasks: tasks/repository.yml
       vars:
         repository_state: present
-    - include_tasks: tasks/repository.yml
+
+    - name: "Create Addititional Repository"
+      ansible.builtin.include_tasks: tasks/repository.yml
       vars:
         repository_state: present
         repository_url: "https://repos.fedorapeople.org/pulp/pulp/demo_repos/zoo/"
-    - include_tasks: tasks/repository.yml
+
+    - name: "Create Container repository"
+      ansible.builtin.include_tasks: tasks/repository.yml
       vars:
         repository_state: present
         repository_name: Test Docker Repository
         repository_content_type: docker
-        repository_url: https://registry.hub.docker.com
-        repository_docker_upstream_name: library/busybox
-    - include_tasks: tasks/katello_sync.yml
-    - include_tasks: tasks/katello_sync.yml
+        repository_url: https://quay.io
```

Comment:
Switched to quay here because Docker Hub limited their api calls, and that breaks it for me 🙂

Reply:
yeah... I've heard people complaining about that